src/SprykerEco/Yves/Computop/CheckoutPage/Process/Steps/PaymentStep.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Computop\CheckoutPage\Process\Steps;
+
+use Spryker\Shared\Kernel\Transfer\AbstractTransfer;
+use Spryker\Shared\Kernel\Transfer\Exception\NullValueException;
+use SprykerShop\Yves\CheckoutPage\Process\Steps\PaymentStep as SprykerShopPaymentStep;
+
+class PaymentStep extends SprykerShopPaymentStep
+{
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return bool
+     */
+    public function isBreadcrumbItemHidden(AbstractTransfer $quoteTransfer): bool
+    {
+        try {
+            $quoteTransfer->getPaymentOrFail()->getComputopPayPalExpressOrFail();
```

Comment:
please get rid from exception driven behavior
you could use `if (!quoteTransfer->getPaymentOrFail() || !$quoteTransfer->getPaymentOrFail()->getComputopPayPalExpressOrFail()) ...`